### ISSUE TITLE
[BUG] Unable to skip tutorials

### ISSUE THREAD

**Issue by notnossravgni** [cupiditys/iReady-Overload#170]
**Describe the bug**
When you're on a tutorial, the extension is unable to skip it. Although it does recognize that it's a lesson, nothing happens once you click "OK" after being warned.

**To Reproduce**
Start a lesson
Load the extension 
Skip the lesson
Click "OK" (the disclaimer that you might get in trouble)
After clicking "OK" nothing happens. I should be kicked out of the lesson/tutorial.

**Expected behavior**
the lesson should be skipped

**Screenshot(s)/Video**
**currently unable to attach a screenshot, will update bug report when i am able to**

**Additional info**

**Comment by cupiditys** [comment]
i can't do anything until u attach video ive skipped tutorials plenty of times

**Comment by notnossravgni** [comment]
https://user-images.githubusercontent.com/77316348/152927783-0a19ead8-531e-47d3-826d-d5b8309b9c77.mp4

alright, heres the video. let me know if anything else is needed

**Comment by cupiditys** [comment]
read what the alerts say bruh "To skip this lesson, close the lesson and click the skip button again."

**Comment by cupiditys** [comment]
send a video

**Comment by notnossravgni** [comment]
> read what the alerts say bruh "To skip this lesson, close the lesson and click the skip button again."

apologize for the late response, but this is what happens when i close out of the tutorial. I also wanted to add on that the same issue happens with "practice" lessons too.

Uploading 18fcf318-fa16-4a52-9e46-df9c27b2e632.mp4…

**Comment by cupiditys** [comment]
it doesn’t even look like you closed the lesson more like you switched tab

**Comment by SharkiArthur** [comment]
yes i noticed, seems like BS works for me. If it is fr just redownlaod to make sure

**Comment by notnossravgni** [comment]
> it doesn’t even look like you closed the lesson more like you switched tab

i got it to work, i just deleted the current bookmarklet and recreated it lmao
